docs/contribute/docker.md

```diff
@@ -21,21 +21,28 @@ docker-compose up
 Use this process if you're a developer and want complete control over
 your Monica container.
 
-Edit `.env` again to set the `DB_*` variables to match your database. Then run:
+If you aren't using docker-compose, edit `.env` again to set the `DB_*` variables to match your database. Then run:
 
 ```sh
-docker build -t monicahq/monicahq .
+# assets are copied from the host machine, make sure they are built
+npm install --production && npm run preproduction && npm run production
```

Comment:
The commands to run are:
```suggestion
yarn install && yarn run production
```

Reply:
I don't think this works: without `preproduction` the build won't complete, unless yarn magically runs that command.

Reply:
Exactly, that's the purpose of this commands: the "pre{pattern}" commands are run before the "{pattern}" command itself
See https://docs.npmjs.com/cli/v7/using-npm/scripts#pre--post-scripts

Reply:
Ah, neat, wasn't aware of that npm feature!